dev/make/deps.mkl.mk

```diff
@@ -17,47 +17,48 @@
 #++
 #  Math backend (MKL) definitions for makefile
 #--
-MKLFPKDIR:= $(if $(wildcard $(DIR)/__deps/mklfpk/$(_OS)/*),$(DIR)/__deps/mklfpk,                        \
-            $(if $(wildcard $(MKLFPKROOT)/include/*),$(subst \,/,$(MKLFPKROOT)),                        \
-            $(error Can`t find MKLFPK libs nether in $(DIR)/__deps/mklfpk/$(_OS) not in MKLFPKROOT.)))
-MKLFPKDIR.include := $(MKLFPKDIR)/include $(MKLFPKDIR)/$(if $(OS_is_fbsd),lnx,$(_OS))/include
-MKLFPKDIR.libia   := $(MKLFPKDIR)/$(if $(OS_is_fbsd),lnx,$(_OS))/lib/$(_IA)
 
-RELEASEDIR.include.mklgpufpk := $(RELEASEDIR.include)/services/internal/sycl/math
+MKLFPKDIR:= $(if $(wildcard $(MKLROOT)/include/*),$(subst \,/,$(MKLROOT)),                        \
+            $(error Can`t find MKLROOT libs nether in $(DIR)/__deps/mklfpk/$(_OS) not in MKLROOT.))
+MKLFPKDIR.include := $(subst \,/,$(MKLFPKDIR)/include)
+MKLFPKDIR.libia   := $(subst \,/,$(MKLFPKDIR)/lib)
+RELEASEDIR.include.mklgpufpk := $(subst \,/,$(RELEASEDIR.include)/services/internal/sycl/math)
 
-MKLGPUFPKDIR:= $(if $(wildcard $(DIR)/__deps/mklgpufpk/$(_OS)/*),$(DIR)/__deps/mklgpufpk/$(_OS),$(subst \,/,$(MKLGPUFPKROOT)))
-MKLGPUFPKDIR.include := $(MKLGPUFPKDIR)/include
-MKLGPUFPKDIR.lib   := $(MKLGPUFPKDIR)/lib/
+MKLGPUFPKDIR:= $(subst \,/,$(MKLFPKDIR))
+MKLGPUFPKDIR.include := $(subst \,/,$(MKLGPUFPKDIR)/include/oneapi)
+MKLGPUFPKDIR.lib   := $(subst \,/,$(MKLGPUFPKDIR)/lib)
 
-mklgpufpk.LIBS_A := $(MKLGPUFPKDIR.lib)/$(plib)daal_sycl$d.$(a)
-mklgpufpk.HEADERS := $(MKLGPUFPKDIR.include)/mkl_dal_sycl.hpp $(MKLGPUFPKDIR.include)/mkl_dal_blas_sycl.hpp
+mklgpufpk.HEADERS := $(subst \,/,$(MKLGPUFPKDIR.include)/mkl.hpp)
+mklgpufpk.LIBS_A := $(subst \,/,$(MKLGPUFPKDIR.lib)/$(plib)mkl_sycl$d.$(a))
 
-daaldep.math_backend.incdir := $(MKLFPKDIR.include) $(MKLGPUFPKDIR.include)
-daaldep.math_backend_oneapi.incdir := $(MKLFPKDIR.include) $(MKLGPUFPKDIR.include)
+daaldep.math_backend.incdir := $(subst \,/,$(MKLFPKDIR.include))
+daaldep.math_backend_oneapi.incdir := $(subst \,/,$(MKLFPKDIR.include) $(MKLGPUFPKDIR.include))
```

Comment:
```suggestion
daaldep.math_backend_oneapi.incdir := $(subst \,/,$(MKLFPKDIR.include) subst \,/,$(MKLGPUFPKDIR.include))
```

Reply:
Its not necessary to subst in this variable at all with new updates